coderadar-server/coderadar-core/src/main/java/org/wickedsource/coderadar/core/rest/dates/series/WeekPoint.java

```diff
@@ -4,24 +4,21 @@
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
 import org.wickedsource.coderadar.core.rest.dates.Week;
 import org.wickedsource.coderadar.core.rest.dates.serialize.WeekDeserializer;
 import org.wickedsource.coderadar.core.rest.dates.serialize.WeekSerializer;
 
 @JsonTypeName("week")
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
same as with `Day`

Reply:
Unfortunately, this doesn't work as @Value makes the fields final, so we can no longer set them using the setters.